pom.xml

```diff
@@ -30,6 +30,20 @@
         <asciidoctorj-version>2.5.2</asciidoctorj-version>
         <jruby-version>9.2.19.0</jruby-version>
 
+        <surefire-version>3.0.0-M5</surefire-version>
```

Comment:
Should be M7

Reply:
Api incompatibility error with M7

Reply:
[08500da](https://github.com/BlackBeltTechnology/osgi-asciidoc-converter/pull/4/commits/08500da679cd5dfed2c00fea990110351f4bdf58)